src/main/resources/application.yaml

```diff
@@ -12,6 +12,12 @@ management:
 spring:
   application:
     name: Blob Router Service
+  datasource:
+    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}${DB_CONN_OPTIONS:}
```

Comment:
do we nee empty value if DB_CONN_OPTIONS is not set. We don't need : ?

Reply:
removed